### ISSUE TITLE
Design GUI for use cases in navigation context area

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#5]
## User story
1. As a User
2. I want to be able to enter an area in the test app where all use cases for navigation are collected
3. So that I can easily switch between the use cases.

I would like to be able to get back to the main view with a "Back" button.

## Acceptance criteria
* In the test app there is a possibility to switch to the navigation context.
* The navigation context is clearly recognizable as such (e.g. by a corresponding title).
* Tapping the "Back" button takes you back to the main view (from which the navigation context was entered).

## Notes
The division into contexts is intended to make the app clearer. In addition to the navigation context, there will later be further contexts for other use case categories.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by i315315** [comment]
Duplicate
